apps/frontend/src/assets/css/scaffolding.css

```diff
@@ -31,3 +31,8 @@ body {
 	white-space: nowrap;
 	border: 0;
 }
+
+label {
+	font-size: 16px;
+	line-height: 120%;
+}
```

Comment:
scaffolding is only for common styles, we will define labels separately where it's needed
```suggestion
```